pkg/cmd/scan/output/testdata/output_stringer_resources.txt

```diff
@@ -1,12 +1,15 @@
 Found missing resources:
```

Comment:
would be a good idea to add a test for a res that have no attribute (HumanMapPrint == "")

Reply:
i add a new test with adapted modification in console.go